tests/acceptance/suites/vertical-full-page-map.js

```diff
@@ -6,12 +6,14 @@ import Pagination from '../blocks/pagination';
 import SearchRequestLogger from '../searchrequestlogger';
 import { VERTICAL_SEARCH_URL_REGEX } from '../constants';
 import { registerIE11NoCacheHook } from '../../test-utils/testcafe';
+import CollapsibleFilters from '../blocks/collapsiblefilters';
 
 fixture`Vertical Full Page Map`
-  .page(`http://localhost:${PORT}/locations_full_page_map`)
+  .page(`http://localhost:${PORT}/locations_full_page_map_with_filters`)
```

Comment:
is there not a lot of differences between `locations_full_page_map` and `locations_full_page_map_with_filters` page? I'm not sure if we should just stop running tests on `locations_full_page_map`.  Does it fail if you have two separate fixtures, uses different url, in the same file?

Reply:
The pages are exactly the same except the `_with_filters` page has filters enabled. All of the map functionality tested on the `locations_full_page_map` page should pass on the `locations_full_page_map_with_filters` page.  I can check if two separate fixtures work in the same file, but I don't see a need for it

Reply:
Multiple fixtures worked fine, but I learned that I needed to re-use the same verticalSearchLogger or else the request logger wouldn't work

Reply:
does the fixtures run concurrently? I'm wondering if using the same verticalSearchLogger could be misleading if one fixture receive response triggered from another fixture and just resolve the waitOnSearchComplete call.

Reply:
The fixtures run sequentially so I don't think that's an issue